napari/layers/points/_tests/test_points.py

```diff
@@ -608,6 +640,100 @@ def test_n_dimensional():
     assert layer.n_dimensional is True
 
 
+@pytest.mark.parametrize("attribute", ['edge', 'face'])
+def test_switch_color_mode(attribute):
+    """Test switching between color modes"""
+    shape = (10, 2)
+    np.random.seed(0)
+    data = 20 * np.random.random(shape)
+    properties = {'point_type': np.array([0, 1.5] * int((shape[0] / 2)))}
```

Comment:
Perhaps this should be `np.random.random`, since it's continuous values?

Reply:
That's a good point. I've updated it so there is a continuous property (using `np.random.random`) and a categorical property.